databricks/koalas/utils/accessors.py

```diff
@@ -0,0 +1,209 @@
+#
+# Copyright (C) 2019 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""
+Utility features. The features here are specific to Koalas.
+"""
+from collections import OrderedDict
+from typing import Tuple, Union, TYPE_CHECKING
+
+from databricks.koalas.internal import InternalFrame, SPARK_INDEX_NAME_FORMAT
+from databricks.koalas.utils import name_like_string, scol_for
+
+if TYPE_CHECKING:
+    from databricks.koalas.frame import DataFrame
+
+
+class UtilsFrameMethods(object):
+    """ Utility features for DataFrame. Usually, The features here are specific to Koalas. """
```

Comment:
Hm, I am less sure if we should add another accessor for this. For example, `apply_batch`, `transform_batch`, `spark` are Koalas specific APIs too.

How about we just expose one method at `DataFrame`? Maybe we could name it like:
- `append_id(type=..., labels=...)`
- `append_id_series(type=..., labels=...)`
- `attach_id_series(type=..., labels=...)`
- `add_id_series(type=..., labels=...)`
- `generate_ids(type=..., labels=...)`
- ...

Reply:
I'd not prefer to have the method at `DataFrame` directly but to separate the namespace since this doesn't exist in pandas DataFrame. For `spark` accessor, as this is less related to Spark, under the accessor might not suitable.

As for the naming and the signature, I'm fine with one of them.

Reply:
I agree with the idea of having a separate accessor. Should we maybe name it `koalas` and put other Koalas specific APIs into it?